test/tests/postprocessors/multi_period_averaging/multi_period_averager.i

```diff
@@ -0,0 +1,79 @@
+[Mesh]
+  [generator]
+    type = GeneratedMeshGenerator
+    dim = 1
+    xmax = 1
+    nx = 100
+  []
+[]
+
+[Problem]
+  type = FEProblem
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [null_kernel]
+    type = NullKernel
+    variable = u
+  []
+[]
+
+[AuxVariables]
+  [v]
+  []
+[]
+
+[AuxKernels]
+  [func_kernel]
+    type = FunctionAux
+    variable = v
+    function = ic
+  []
+[]
+
+[Functions]
+  [ic]
+    type = ParsedFunction
+    expression = '-4 * x * (x-1)'
```

Comment:
This change was made and the test was regolded to address this suggestion.

Reply:
Marking this as unresolved because this change seemed to have been applied to `test/tests/postprocessors/periodic_integration/periodic_integral.i` instead of this test file.

Reply:
The test was changed back to the original form for `periodic_integration` and the `multi_period_averager` was updated to include this suggestion.